### ISSUE TITLE
Unsupported ARIA attribute - Accessibility Issue 

### ISSUE THREAD

**Issue by ghost** [valor-software/ng2-bootstrap#6503]
using    "ngx-bootstrap": "^5.3.2"

ngbTypeAhead autogenerated code violates the following accessibility rules. The code should NOT generate **aria-multiline="false"** when role="combobox"

_ARIA attribute is not allowed: aria-multiline="false"_

https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome

here is a snippet of generated code

`<input _ngcontent-ng-cli-universal-c6="" aria-multiline="false" autocapitalize="off" autocorrect="off" class="form-control ng-untouched ng-pristine ng-valid" formcontrolname="xxxxxxxxx" id="xxxxxxxxxx" role="combobox" tabindex="0" type="text" autocomplete="off" aria-autocomplete="list" aria-expanded="false">`
